README.md

```diff
@@ -402,6 +402,15 @@ Vert.x Mutiny webClient exploratory test.
 
 Also see http/vertx-web-client/README.md
 
+### `http/vertx-web-validation`
+Ensure that you can deploy a simple Quarkus application with predefined routes using schema parser from vertx-json-schema and Router Vert.x approach,
+ incorporating web validation configuration through ValidationHandlerBuilder. One of the goal of vertx-web-validation functionality is to validate parameters and bodies of incoming requests.
+
+It also verifies multiple deployment strategies like:
+- Serverless
```

Comment:
Well, that was a mistake "copy paste README sections", but I guess I would use on an `@OpenShiftScenario(deployment = OpenShiftDeploymentStrategy.UsingOpenShiftExtension) `
including this profile:
` <profile>
            <id>deploy-to-openshift-using-extension</id>
            <dependencies>
                <dependency>
                    <groupId>io.quarkus</groupId>
                    <artifactId>quarkus-openshift</artifactId>
                </dependency>
            </dependencies>
        </profile>`
        And exclude the ARM OS because https://github.com/quarkus-qe/quarkus-test-suite/issues/1142 
        Anyways, I agree with you that this is not necessary to implement it :).

Reply:
Anyways, would I be in the right way?